design-library/src/components/BccGraphic/BccGraphic.vue

```diff
@@ -0,0 +1,26 @@
+<script setup lang="ts">
+import { OpenInNewIcon, CheckCircleFillIcon } from "@bcc-code/icons-vue";
+import BccPoster from "./BccPoster.vue";
+
+type Props = {
+  bannerSrc?: string;
+  logoSrc?: string;
+  checked?: boolean;
+  linkOut?: boolean;
+  rounding: "md" | "xl";
```

Comment:
The first two are booleans, but it can be nice to add a default of `false` below, primarily so Storybook shows it (but also because it will otherwise be `undefined`, which is falsy but not exactly `false`).

On `rounding`, it feels counterintuitive to me that the only options are `md` and `xl`. Since `xl` is the default, I would rather expect a `base` size, and then from there `sm` or `lg`. See the button for example.

Lastly, rounding has a default so it should also be optional I think.